aiohttp/_websocket/reader_py.py

```diff
@@ -469,6 +470,6 @@ def parse_frame(
                 self._frame_payload_len = 0
                 self._state = READ_HEADER
 
-        self._tail = buf[start_pos:] if start_pos < buf_length else b""
+        self._tail = buf_cstr[start_pos:buf_length] if start_pos < buf_length else b""
```

Comment:
I think it looks OK to me, but maybe add a comment highlighting that we can't use an implicit end here to ensure nobody refactors it later.
```suggestion
        # XXX: Cython needs slices to be bounded, so we can't omit the slice end here.
        self._tail = buf_cstr[start_pos:buf_length] if start_pos < buf_length else b""
```